include/uv.h

```diff
@@ -260,7 +260,8 @@ typedef struct uv_metrics_s uv_metrics_t;
 
 typedef enum {
   UV_LOOP_BLOCK_SIGNAL = 0,
-  UV_METRICS_IDLE_TIME
+  UV_METRICS_IDLE_TIME,
+  UV_LOOP_USE_IO_URING_SQPOLL
```

Comment:
```suggestion
  UV_LOOP_USE_IO_URING_SQPOLL
#define UV_LOOP_USE_IO_URING_SQPOLL UV_LOOP_USE_IO_URING_SQPOLL
```
So users can ifdef-guard the feature like this:
```c
#ifdef UV_LOOP_USE_IO_URING_SQPOLL
uv_loop_configure(loop, UV_LOOP_USE_IO_URING_SQPOLL, 1);
#endif
```